web/src/params/user.rs

```diff
@@ -0,0 +1,51 @@
+use sea_orm::Value;
+use serde::Deserialize;
+use utoipa::{IntoParams, ToSchema};
+
+use domain::{IntoUpdateMap, UpdateMap};
+
+#[derive(Debug, Deserialize, IntoParams, ToSchema)]
+pub struct UpdateUserParams {
+    pub email: Option<String>,
+    pub first_name: Option<String>,
+    pub last_name: Option<String>,
+    pub display_name: Option<String>,
+    pub github_profile_url: Option<String>,
+}
+
+impl IntoUpdateMap for UpdateUserParams {
```

Comment:
I love this pattern for updating just the right fields - it's extremely clean, easy to understand and effective.